charts/pmp-reactjs-ui/templates/configmap.yaml

```diff
@@ -11,7 +11,8 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  REACT_APP_API_BASE_URL: https://{{ .Values.pmp_new.react_app_api_base_url }}/v1/partnermanager
+  REACT_APP_PARTNER_MANAGER_API_BASE_URL: https://{{ .Values.pmp_new.react_app_partner_manager_api_base_url }}/v1/partnermanager
```

Comment:
since variable reads base url. Why cant we have the value with the http or https being pulled directly from values?